sunpy/time/__init__.py

```diff
@@ -5,3 +5,4 @@
 from sunpy.time.timerange import *
 from sunpy.time.julian import *
 from sunpy.time.utime import *
+from sunpy.time import astropy_time
```

Comment:
I would change this to `from sunpy.time.astropy_time import Time` so you can just `from sunpy.time import Time` and then when we retire the subclass it's just a `s/sunpy.time/astropy.time`